win32/src/gballoc_ll_jemalloc.c

```diff
@@ -214,3 +218,97 @@ void gballoc_ll_print_stats(void)
+
+                    /* Codes_SRS_GBALLOC_LL_JEMALLOC_28_005: [ For each existing arena: ]*/
+                    for (uint64_t i = 0; i < narenas; i++) {
+                        int snprintf_result = snprintf(command, sizeof(command), "arena.%" PRIu64 ".dirty_decay_ms", i);
+
+                        if (snprintf_result < 0)
+                        {
+                            /* Codes_SRS_GBALLOC_LL_JEMALLOC_28_008: [ If there are any errors, gballoc_ll_set_decay shall fail and return a non-zero value. ]*/
+                            LogError("snprintf failed for arena %" PRIu64 "", i);
+                            result = MU_FAILURE;
+                        }
+                        else
+                        {
+                            /* Codes_SRS_GBALLOC_LL_JEMALLOC_28_006: [ gballoc_ll_set_decay shall set the dirty decay time for the arena to decay_milliseconds milliseconds. ]*/
+                            if (je_mallctl(command, NULL, NULL, &decay_milliseconds, sizeof(decay_milliseconds)) != 0)
+                            {
+                                /* Codes_SRS_GBALLOC_LL_JEMALLOC_28_008: [ If there are any errors, gballoc_ll_set_decay shall fail and return a non-zero value. ]*/
+                                LogError("failed to set dirty decay time for arena %" PRIu64 "", i);
+                                result = MU_FAILURE;
+                                break;
+                            }
+
+                            snprintf_result = snprintf(command, sizeof(command), "arena.%" PRIu64 ".muzzy_decay_ms", i);
+
+                            if (snprintf_result < 0)
+                            {
+                                /* Codes_SRS_GBALLOC_LL_JEMALLOC_28_008: [ If there are any errors, gballoc_ll_set_decay shall fail and return a non-zero value. ]*/
+                                LogError("snprintf failed for arena %" PRIu64 "", i);
+                                result = MU_FAILURE;
+                            }
+                            else
+                            {
+                                /* Codes_SRS_GBALLOC_LL_JEMALLOC_28_007: [ gballoc_ll_set_decay shall set the muzzy decay time for the arena to decay_milliseconds milliseconds. ]*/
+                                if (je_mallctl(command, NULL, NULL, &decay_milliseconds, sizeof(decay_milliseconds)) != 0)
+                                {
+                                    /* Codes_SRS_GBALLOC_LL_JEMALLOC_28_008: [ If there are any errors, gballoc_ll_set_decay shall fail and return a non-zero value. ]*/
+                                    LogError("failed to set muzzy decay time for arena %" PRIu64 "", i);
+                                    result = MU_FAILURE;
+                                    break;
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    return result;
+}
```

Comment:
Because I am asking questions in the review, I need to followup with the answers, so I shall be waiting.

Reply:
I've briefly looked at iteration 6, and I cannot notice any INT tests that would confirm behavior.

I strongly advise to have INT tests before all this can be checked in.

Reply:
Sure I will add the INT tests as well in this PR.